src/server/input_service.rs

```diff
@@ -379,8 +379,8 @@ fn handle_mouse_(evt: &MouseEvent, conn: i32) {
     let mut to_release = Vec::new();
     if evt_type == 1 {
         fix_modifiers(&evt.modifiers[..], &mut en, 0);
-        #[cfg(target_os = "macos")]
-        en.reset_flag();
+        // #[cfg(target_os = "macos")]
+        // en.reset_flag();
```

Comment:
Can I know how reset_flag causes the problem?

Reply:
The command modifier flag should not be zeroed when the mouse is pressed or raised. If it's an old code, when the second mouse click is pressed, the flag will be already zeroed.
This flag should be zeroed when the command key is raised.

Reply:
Yes, it causes bug for controlling mac with mobile, mobile does not send key up event to mac.